### ISSUE TITLE
Debian package support

### ISSUE THREAD

**Issue by ampledata** [ProCern/asyncinotify#6]
Thank you for using [PEP-517](https://peps.python.org/pep-0517/) with this project. Unfortunately I haven't been able to find any plug-and-play tooling to allow Debian package creation from PEP-517 projects. My existing solution was to use [stdeb](https://github.com/astraw/stdeb), but it doesn't support PEP-517 (and I'm not sure if it ever will?). That said, I've forked this repo over at [snstac](https://github.com/snstac/asyncinotify) and will attempt to shim support in.

Don't feel obliged to do any work on this - as Debian package support is definitely self-serving for me, but if you have any insights it would be greatly appreciated.

**Comment by TaylorRichberger** [comment]
Yeah, I haven't done much with that kind of stuff.  All my Debian package creation work has been either manual, using the standard Debian package tools (and so lightly and infrequently there that I have to relearn the whole thing every time I touch it) or using CMake's CPack for the C++ projects I have to work with.  Personally, I'd probably just do it manually using the Debian packaging guides and examining other Debian Python packages, trying to find another one based on PEP-517 to see how they work around it.
